sops/3scale-scaling.md

```diff
@@ -0,0 +1,65 @@
+# 3scale scaling on Openshift 3.11, Integreatly 1.4.x
+This guide states the known limits of 3scale capacity on Openshift along with scaling options for 3scale Components.
+
+**Note :** Pod anti-affinity rule is in place across a number of 3scale components. This may result in limited scaling options.
+
+## 3scale Scaling Components
+**Note :** Components are ordered in priority of known bottle necks.
+
+
+1. Backend-Listener
+    - Pod **anti-affinity** rule is important here
+    - You can scale this component **horizontally** by adding more PODs
```

Comment:
Would be useful to provide a sample one liner `oc` command for scaling the pod count of this component. Something that the SRE team could almost copy/paste